superset/views/base_api.py

```diff
@@ -82,6 +82,18 @@ class BaseSupersetModelRestApi(ModelRestApi):
         }
     """  # pylint: disable=pointless-string-statement
 
+    def _init_properties(self):
```

Comment:
Would it be possible to set the default column list to `[]` (empty array) to have nothing returned by default? Database primary keys can be sensitive under certain circumstances.

Reply:
Everything is possible, on this case would probably involve a somewhat delicate change on FAB. My take is that, this is just a fallback, enough to force everyone to always declare CRUD fields explicitly or else the API will return nothing of value (like just the id's)

Reply:
I think being able to default to fully-closed is important. This isn't a blocker for this PR, but it's a change I would really like to see on FAB.